core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/candlestickChart/CandlestickChartOpDesc.scala

```diff
@@ -0,0 +1,91 @@
+package edu.uci.ics.texera.workflow.operators.visualization.candlestickChart
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, AttributeType, Schema}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.operators.visualization.{
+  VisualizationConstants,
+  VisualizationOperator
+}
+
+class CandlestickChartOpDesc extends VisualizationOperator with PythonOperatorDescriptor {
+
+  @JsonProperty(value = "date", required = true)
+  @JsonSchemaTitle("Date Column")
+  @JsonPropertyDescription("the date of the candlestick")
+  @AutofillAttributeName
+  var date: String = ""
+
+  @JsonProperty(value = "open", required = true)
+  @JsonSchemaTitle("Open Column")
+  @JsonPropertyDescription("the opening value of the candlestick")
+  @AutofillAttributeName
+  var open: String = ""
```

Comment:
Please change the "Open", "High", "Low", "Close" to more accurate name. For example, "open column" should be "opening price column" for accurate expression.